core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -851,6 +851,14 @@ public enum Property {
           + " the tablet server this duration to revive before reassigning its tablets"
           + " to other tablet servers."),
 
+  TABLE_LOCATION_MODE("table.location.mode", "locality", PropertyType.LAST_LOCATION_MODE,
```

Comment:
Do we want this to be a per-table property, or a global property?

Reply:
I thought a per table propery might be useful, but perhaps not.  I can go either way on this one.

Reply:
@ctubbsii Is there a reason so make this a _general_ property versus a _tserver_ property?  We are talking about tserver locations so that would have been be my first inclination.

Reply:
A tserver property would make sense.